terraform/modules/daily_snapshot/service/daily_snapshot.rb

```diff
@@ -63,6 +74,9 @@ def latest_snapshot_date(chain_name = 'calibnet')
   end
 end
 
+# Prune logs files(txt) in the logs directory older than 2 weeks
+prune_logs
+
```

Comment:
Let's do it at the end of the file. If it fails for whatever reason, at least the rest of the script will finish.

Reply:
@LesnyRumcajs done